src/HeadProvider.js

```diff
@@ -66,6 +66,14 @@ export default class HeadProvider extends React.Component {
     },
   };
 
+  componentDidUpdate() {
+    const ssrTags = document.head.querySelectorAll(`[data-rh=""]`);
+
+    for (let i = 0; i < ssrTags.length; i += 1) {
```

Comment:
Could we just forEach here? `ssrTags.forEach(t => t.remove());`

Reply:
Sure

Reply:
However, do you want to support ie11?

Reply:
Or we can support it with the request.